src/inventory.cc

```diff
@@ -2359,42 +2361,21 @@ static void _inven_pickup(int buttonCode, int indexOffset)
         _display_inventory(indexOffset, itemIndex, INVENTORY_WINDOW_TYPE_NORMAL);
     }
 
-    FrmImage itemInventoryFrmImage;
-    int itemInventoryFid = itemGetInventoryFid(item);
-    if (itemInventoryFrmImage.lock(itemInventoryFid)) {
-        int width = itemInventoryFrmImage.getWidth();
-        int height = itemInventoryFrmImage.getHeight();
-        unsigned char* data = itemInventoryFrmImage.getData();
-        mouseSetFrame(data, width, height, width, width / 2, height / 2, 0);
-        soundPlayFile("ipickup1");
-    }
-
     if (itemInHand != nullptr) {
         _inven_update_lighting(nullptr);
     }
 
-    do {
-        sharedFpsLimiter.mark();
-
-        inputGetInput();
-        _display_body(-1, INVENTORY_WINDOW_TYPE_NORMAL);
-
-        renderPresent();
-        sharedFpsLimiter.throttle();
-    } while ((mouseGetEvent() & MOUSE_EVENT_LEFT_BUTTON_REPEAT) != 0);
-
-    if (itemInventoryFrmImage.isLocked()) {
-        itemInventoryFrmImage.unlock();
-        soundPlayFile("iputdown");
-    }
+    // SFALL: allow ctrl-click to unequip
```

Comment:
Not sure if "SFALL" reference is correct here if the feature is not the same.  Also I understand this is not just for unequip.

Reply:
In this function, it's just ctrl-click to unequip, which is the same as sfall, no?  It doesn't have the auto-equip part of the feature.  I'm open to adding that if people think it's important.  It's just a little finicky in the CE code since we can't jump straight into `if` bodies like sfall can.